localstack-core/localstack/services/events/provider.py

```diff
@@ -1152,7 +1153,16 @@ def put_targets(
         failed_entries = rule_service.add_targets(targets)
         rule_arn = rule_service.arn
         rule_name = rule_service.rule.name
-        for target in targets:  # TODO only add successful targets
+        for index, target in enumerate(targets):  # TODO only add successful targets
```

Comment:
came too late, but python tip: `enumerate` takes a `start` argument, avoiding this `index+1` issue

`for index, target in enumerate(targets, start=1):`

Reply:
That is really cool, didn't know that!